### ISSUE TITLE
Donations are hanging for one donor

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/giveth-dapp#438]
https://beta.giveth.io/donations

<img width="1267" alt="screen shot 2018-07-22 at 6 45 35 pm" src="https://user-images.githubusercontent.com/14945613/43048004-b7d6781e-8de0-11e8-8d3a-8c08006c43f3.png">

3 donations are hanging... one to a milestone which needs to be canceled (I already re-donated and it worked! ) 2 went to different campaigns... those could be pushed thru... ;-)

Thx guys

**Comment by GriffGreen** [comment]
oops wrong screen shot

**Comment by GriffGreen** [comment]
<img width="1159" alt="screen shot 2018-07-22 at 6 59 57 pm" src="https://user-images.githubusercontent.com/14945613/43048040-7a678850-8de1-11e8-87b1-97f30e5f8642.png">

**Comment by perissology** [comment]
yes, this is a know bug regarding delegations that are past their commit time. It's the same bug that is blocking bowens milestone from being completed.

working on a fix
